index.html

```diff
@@ -1,165 +1,232 @@
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="tags">
+          <li class="tags-elements">html</li>
+          <li class="tags-elements">bootstrap</li>
+          <li class="tags-elements">Ruby</li>
+        </ul>
+        <a class="button" href="">See Project</a>
+      </section>
+    </section>
+    <section class="about">
+      <h1>About me</h1>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a class="button" href="">Get My Resume</a>
+      <span class="horizontal-line"></span>
+      <ul class="about-list">
+        <li id="diamond">
+          Languages
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </li>
+        <li id="square">
+          Frameworks
+          <ul>
+            <li>bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CopyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li id="circle">
+          Skills
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section class="contact-form-section">
+      <h1 class="contact-description">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h1>
+      <form class="contact-form" action="https://formspree.io/f/mqknarzz" method="POST">
+        <input
+        class="name"
+          type="text"
+          name="username"
+          placeholder="Fullname"
+          maxlength="30"
+          required="required"
+        />
+        <input
+        class="email"
+          type="email"
+          name="email"
+          placeholder="Email address"
+          required="required"
+        />
+        <textarea
+        class="message"
+          name="message"
+          placeholder="Enter text here"
+          maxlength="500"
+          required="required"
+        ></textarea>
+        <button class="form-button" type="submit">Get in touch</button>
+      </form>
+      <ul class="icons-container">
+        <li>
+          <a href=""><span class="github-icon-dark"></span></a>
+        </li>
+        <li>
+          <a href=""><span class="linkedin-icon-dark"></span></a>
+        </li>
+        <li>
+          <a href=""><span class="fingers-icon-dark"></span></a>
+        </li>
+        <li>
+          <a href=""><span class="twitter-icon-dark"></span></a>
+        </li>
+        <li>
+          <a href=""><span class="m-icon-dark"></span></a>
+        </li>
+      </ul>
```

Comment:
- [ ] Add this bar at the end of the icons social media :point_down: 

- Your design:

![111](https://user-images.githubusercontent.com/57562869/148842380-9ec8a35b-5585-4d71-859d-649d0eef0eac.png)

- The Original : 
![222](https://user-images.githubusercontent.com/57562869/148842441-ccf396cc-0004-4026-a6eb-5b780327cf11.png)